class-two-factor-core.php

```diff
@@ -1842,15 +1878,44 @@ public static function user_two_factor_options( $user ) {
 			<p><?php echo wp_kses_post( $notice ); ?></p>
 		</div>
 		<?php endforeach; ?>
+
+		<fieldset id="two-factor-options" <?php echo $show_2fa_options ? '' : 'disabled="disabled"'; ?>>
+		<?php
+		if ( $providers ) {
+			self::render_user_providers_form( $user, $providers );
```

Comment:
To avoid wrapping a large block into this conditional, I've moved the form to a helper method.